### PR TITLE
Hosted private beta 2 and 3 t and c on different paths

### PR DESCRIPTION
**Pull request by liamwilson-kainos**

### CODE REVIEW

File: app/views/confirmation.njk [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     <p class="govuk-body">We have sent you a confirmation email and your agreement summary.</p>
 
-    <p class="govuk-body">The agreement will be governed by the agreed <a href="/apply/terms">terms and conditions</a></p>
+    <p class="govuk-body">The agreement will be governed by the agreed <a href="{{latestTermsAndConditionsUri}}" target="_blank">terms and conditions</a></p>
```
Comment (chrissmKainos) [inline app/views/confirmation.njk:17]: any reason this is now going to a new window?

File: app/views/declaration.njk [hunk 2]
```diff
@@ -40,7 +40,7 @@
       <li>you’re the registered keeper</li>
       <li>you’re applying for a review on a CPH registered in England</li>
       <li>{{ minNumText }}</li>
-      <li>you’ve carefully read and accept the <a href="/apply/terms">full terms and conditions</a></li>
+      <li>you’ve carefully read and accept the <a href="{{latestTermsAndConditionsUri}}" target="_blank">full terms and conditions</a></li>
```
Comment (chrissmKainos) [inline app/views/declaration.njk:43]: Same as previous comment, is this a new requirement to open in a new tab?

File: app/views/confirmation.njk (hunk 1 above)
Comment (liamwilson-kainos) [reply app/views/confirmation.njk:17]: Bit of poetic license 👀  seems weird to me that you would replace a confirmation screen with T and C and then you cannot get back to it. Thought a new tab would be more appropriate

**Review by chrissmKainos — DISMISSED** [review]

File: app/views/declaration.njk (hunk 2 above)
Comment (liamwilson-kainos) [reply app/views/declaration.njk:43]: removed as per https://design-system.service.gov.uk/styles/typography/#links:~:text=link%20icon.-,Opening%20links%20in%20a%20new%20tab,-Avoid%20opening%20links

File: app/views/confirmation.njk (hunk 1 above)
Comment (liamwilson-kainos) [reply app/views/confirmation.njk]: removed as per

https://design-system.service.gov.uk/styles/typography/#links:~:text=link%20icon.-,Opening%20links%20in%20a%20new%20tab,-Avoid%20opening%20links

**Review by chrissmKainos — DISMISSED** [review]

**Review by chrissmKainos — APPROVED** [review]

**Pull request merged**